internal/cli/auth/login.go

```diff
@@ -49,20 +51,119 @@ type LoginConfig interface {
 	ProjectID() string
 }
 
+type TrackAsker interface {
```

Comment:
while I think this is fine, why have you added this? don't we just disable telemetry on our tests?